docs/source/examples/VOC2012Segmentation.rst

```diff
@@ -0,0 +1,192 @@
+######################
+ VOC2012 Segmentation
+######################
+
+This topic describes how to manage the `VOC2012 Segmentation dataset`_,
+which is a dataset with :ref:`reference/label_format/SemanticMask:SemanticMask`
+and :ref:`reference/label_format/InstanceMask:InstanceMask` labels
+(:numref:`Fig. %s <example-semantic-mask>` and :numref:`Fig. %s <example-instance-mask>`).
+
+.. _VOC2012 Segmentation dataset: https://gas.graviti.cn/dataset/hello-dataset/VOC2012Segmentation
+
+.. _example-semantic-mask:
+
+.. figure:: ../images/example-semanticmask.png
+   :scale: 150 %
+   :align: center
+
+   The preview of a semantic mask from "VOC2012 Segmentation".
+
+.. _example-instance-mask:
+
+.. figure:: ../images/example-instancemask.png
+   :scale: 150 %
+   :align: center
+
+   The preview of a instance mask from "VOC2012 Segmentation".
+
+
+*****************************
+ Authorize a Client Instance
+*****************************
+
+An :ref:`reference/glossary:accesskey` is needed to authenticate identity when using TensorBay.
+
+.. literalinclude:: ../../../docs/code/VOC2012Segmentation.py
+   :language: python
+   :start-after: """Authorize a Client Instance"""
+   :end-before: """"""
+
+****************
+ Create Dataset
+****************
+
+
+.. literalinclude:: ../../../docs/code/VOC2012Segmentation.py
+   :language: python
+   :start-after: """Create Dataset"""
+   :end-before: """"""
+
+******************
+ Organize Dataset
+******************
+
+It takes the following steps to organize "VOC2012 Segmentation" dataset by the :class:`~tensorbay.dataset.dataset.Dataset` instance.
+
+Step 1: Write the Catalog
+=========================
+
+A :ref:`Catalog <reference/dataset_structure:catalog>` contains all label information of one dataset,
+which is typically stored in a json file.
+
+.. literalinclude:: ../../../tensorbay/opendataset/VOC2012Segmentation/catalog.json
+   :language: json
+   :name: voc2012segmentation-catalog
+   :linenos:
+
+The annotation types for "VOC2012 Segmentation" are :ref:`reference/label_format/SemanticMask:SemanticMask`
+and :ref:`reference/label_format/InstanceMask:InstanceMask`, and there are 22
+:ref:`reference/label_format/CommonLabelProperties:Category` types for :ref:`reference/label_format/SemanticMask:SemanticMask`. There are 2
+:ref:`reference/label_format/CommonLabelProperties:Category` types for :ref:`reference/label_format/InstanceMask:InstanceMask`, category 0 represents the
+background, category 255 represents the border of instances, and other different categories represent different instances.
```

Comment:
There are only two categories in instance mask. What does other different category mean?